### PR TITLE
check cluster-manager exist when apply global set ns

### PR DESCRIPTION
**Pull request by ldpliu**
Signed-off-by: ldpliu <daliu@redhat.com>

https://github.com/stolostron/backlog/issues/24532

### CODE REVIEW

**Comment by ldpliu** [comment]
/hold 
need https://github.com/stolostron/backplane-operator/pull/252 merged

File: pkg/controllers/clusterset/globalclusterset/globalset_controller.go [hunk 1]
```diff
@@ -22,6 +22,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	globalNamespaceAnnotation = "globalClusterSetNamespace"
```
Comment (qiujian16) [inline pkg/controllers/clusterset/globalclusterset/globalset_controller.go:26]: open-cluster-management.io/ns-create

Comment (ldpliu) [reply pkg/controllers/clusterset/globalclusterset/globalset_controller.go]: done

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**